### ISSUE TITLE
[Adaptive Cards> Designer> Select host app]: Screen reader is not announcing the 'State' for the 'Select host app' combobox.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7916]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25197.1000)
Browser: Edge Dev (107.0.1379.1)
URL: https://adaptivecards.io/designer/
Screen reader: Narrator

**Repro steps:**
1. Open the above URL in edge dev browser and login with v-.
2. Turn on the narrator through 'Ctrl+ Windows+ Enter key.
3. Navigate to the 'Designer' tab through tab key and activate it.
4. Navigate to the 'Select host app' combo box through tab key.
5. Observe the issue.

**Actual Result:**
On Designer tab, Screen reader is not announcing the 'Expand/Collapse' state for 'Select host app' combo box. Screen reader is announcing as **'Select host app Bot framework web chat combo box has pop-up'.**

**Expected Result:**
On Designer tab, Screen reader should announce the 'Expand/Collapse' state for 'Select host app' combo box. Screen reader should announce as **'Select host app Bot framework web chat combo box collapsed' when it is in collapsed state and expanded when it is expanded state.**

**Notes:**
1. Same issue repro with 'Target Version' combo box.

**Observation:**
Same issue repro with NVDA and JAWS.

**User Impact:**
Screen reader user will get impacted as user will not be able to understand that combo box is when on expanded or in collapsed state due to which user will have difficulty in making track of the content properly.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104483005/191715470-44397130-d842-4434-9645-739313f3dbf5.mp4

<img width="960" alt="Code Snipped~ State is not announced by narrator" src="https://user-images.githubusercontent.com/104483005/191717804-efd8a64d-1391-4d05-839e-8f819c2bd050.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is working fine on the environment below, hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25330.1000)
Browser Version: 114.0.1807.1
